opentelemetry-sdk/setup.cfg

```diff
@@ -53,6 +53,9 @@ opentelemetry_tracer_provider =
     sdk_tracer_provider = opentelemetry.sdk.trace:TracerProvider
 opentelemetry_propagator =
     b3 = opentelemetry.sdk.trace.propagation.b3_format:B3Format
+opentelemetry_exporter =
+    console_span = opentelemetry.sdk.trace:ConsoleSpanExporter
+    console_metrics = opentelemetry.sdk.metrics:ConsoleMetricsExporter
```

Comment:
This fails when testing locally, with the following error:

```
ImportError: module 'opentelemetry.sdk.trace' has no attribute 'ConsoleSpanExporter'
```

Currently, there's no `ConsoleSpanExporter` import in `opentelemetry.sdk.trace.__init__.py`, so the line should be changed to:

```
console_span = opentelemetry.sdk.trace.export:ConsoleSpanExporter
```

This is not required for the metrics one, as `ConsoleMetricsExporter` exists in `opentelemetry.sdk.metrics.__init__.py`, but could be changed too, to avoid issues in the future.

Reply:
I think the issue exists for [ConsoleMetricsExporter](https://github.com/open-telemetry/opentelemetry-python/blob/master/opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/__init__.py#L71) as well so that would also have to be changed.

Reply:
The changes have been made. @lzchen could you please review and let me know your thoughts?